projects/ui/src/components/Common/Form/TokenInputField.tsx

```diff
@@ -52,6 +57,10 @@ export type TokenInputCustomProps = {
    *
    */
   allowNegative?: boolean;
+  /**
+   * whether to use the wrapped variant or the default text field variant
+   */
+  inputVariant?: 'default' | 'wrapped'
```

Comment:
Do we need to add support for the old variant? Wondering if we can just pull this and have the new `wrapped` version propagate across the site.

Reply:
Yup! Can do!